client/reader/recent/style.scss

```diff
@@ -155,13 +155,13 @@
 			.recent-feed__list-column-header {
 				max-width: none;
 				margin: 0;
-				padding: 16px 0;
+				padding: 16px 24px;
```

Comment:
Moved the left-right padding up a level to prevent too much padding at larger screen sizes.